### ISSUE TITLE
"python3": executable file not found in $PATH: unknown

### ISSUE THREAD

**Issue by yuubd** [delimitrou/DeathStarBench#122]
When I enter docker-compose up -d on socialNetwork directory, I get this error.

Error response from daemon: OCI runtime create failed: container_linux.go:367: starting container process caused: exec: "python3": executable file not found in $PATH: unknown

When I modified python3 -> python in docker-compose.yml tried to run, I get this error.
python: can't open file './scripts/config.py': [Errno 2] No such file or directory

I do a small hack to see what is in scripts by running 
$ sudo docker run --rm --entrypoint ls yg397/social-network-microservices:latest ./scripts, then I see
init_redis_cluster.py
init_social_graph.py

**Comment by iideas18** [comment]
Hi bro, have you fix this problem?

**Comment by kiiimes** [comment]
Hi, I have the same problem. How did fix this problem?

**Comment by Stvdputten** [comment]
The latest dockerhub build of yg397 doesn't include python3 (and some other files), so you need to build the docker images yourself from within the /docker directory. 

If you wish to try out the docker-compose.yml you can also try my build, where you need to replace yg397/image:tag with stvdputten/social-network-microservices:latest or stvdputten/openresty-thrift:latest . I hope that helps.

**Comment by yuubd** [comment]
Sorry guys, I did not see the notification. Yes, the issue is solved.
The problem was that the images are not up to date and I do not think it is being maintained.
I was creating my own images but thanks to @zhxie, I was able to run the application.
These are my notes about the steps:

Issue: Two important images (yg397/thrift-microservice-deps:xenial and  yg397/social-network-microservices) from docker hub are not updated. So, need to build them from the code.

- Build `yg397/thrift-microservice-deps:xenial` image
1. cd into thrift-microservice-deps/cpp
2. `docker build -t yg397/thrift-microservice-deps:xenial .`
3. Go back to the root social network directory
5.`docker build -t yg397/social-network-microservices:latest .`
6. Now run `docker-compose up -d

Let me know if it still does not work.
Make sure to run it in Linux. It somehow does not work on MacOS.
